src/Clapi/RelayApi.hs

```diff
@@ -86,8 +89,8 @@ relayApiProto ownerMv selfAddr =
         ("owner_info", tdp), ("owners", adp), ("build", tdp)]
       , arrayDefMsg catp "clientsdoc" citp
       , Right $ UMsgSetChildren cap [ownSeg] Nothing
-      , staticAdd (cap +| ownSeg) []
-      , tupleDefMsg citp "client info" []
+      , staticAdd (cap +| ownSeg) [tdClVal tdZero]
+      , tupleDefMsg citp "client info" [("clock_diff", "float")]
```

Comment:
This is the perfect chance to start adding docstrings that actually mean something. What units is the clock diff in? ;-)